src/app/pages/form/dropdown/dropdown.component.html

```diff
@@ -37,18 +34,15 @@
       open-direction="down"
 
     >
-      <ng-container *ngFor="let town of nowegianTowns; let idx = index">
-        <tds-dropdown-option [value]="town.value">
-          {{ town.label }}
-        </tds-dropdown-option>
-      </ng-container>
+      <tds-dropdown-option *ngFor="let town of nowegianTowns; let idx = index" value="{{town.value}}">
```

Comment:
Nice with the value in the *ngFor 🍻

Reply:
Yes, ng-container is not needed here. :)